Source/Scene/Cesium3DTile.js

```diff
@@ -553,11 +558,18 @@ define([
      */
     Cesium3DTile.prototype.createBoundingVolume = function(boundingVolumeHeader, transform, result) {
         var center;
+        var hasTileTransform = !Matrix4.equals(this.computedTransform, Matrix4.IDENTITY);
         if (defined(boundingVolumeHeader.box)) {
             var box = boundingVolumeHeader.box;
             center = Cartesian3.fromElements(box[0], box[1], box[2], scratchCenter);
             var halfAxes = Matrix3.fromArray(box, 3, scratchHalfAxes);
 
+            if (hasTileTransform) {
+                // Transform from y-up to z-up
+                Matrix3.multiply(yUpToZUpMatrix3, halfAxes, halfAxes);
+                Matrix4.multiplyByPoint(yUpToZUpMatrix4, center, center);
+            }
```

Comment:
To be consistent with glTF, `y` should be up.

To be consistent with WGS84 (and local east-north-up), `z` should be up.

If I understand you correctly, to be consistent with local vs WGS84 coordinates, `z` should be up.

It would be nice to be consistent with glTF, but it sounds like we have a compelling reason to go with `z` up.  Do you agree?